code/tailer/datastripe.q

```diff
@@ -124,28 +143,27 @@ savetablesoverperiod:{[dir;tablename;nextp;lasttime]
 
     /- enumerate and then split by keycol
     symdir:` sv dir,.proc.procname;
-    enumkeycol: .Q.en[symdir;?[tablename;enlist (<;`time;nextp);0b;()]];
+    enumkeycol: .Q.en[symdir;eval tablename];
```

Comment:
is there a reason to use `eval` here? I think `value` would be more conventional

Reply:
Yeh, I am not sure how I used eval here..